charts/nexus-repository-manager/README.md

```diff
@@ -10,53 +10,83 @@
+- Integrate with industry-leading build tools.
+---
 
 ## Introduction
 
-This chart bootstraps a Nexus OSS deployment on a cluster using Helm.
+This chart installs a single Nexus Repository instance within a Kubernetes cluster that has a single node (server) configured. It is not appropriate for a resilient Nexus Repository deployment. Refer to our [resiliency documentation](https://help.sonatype.com/repomanager3/planning-your-implementation/resiliency-and-high-availability) for information about resilient Nexus Repository deployment options.
 
-## Prerequisites
+Use the checklist below to determine if this Helm chart is suitable for your deployment needs.
+
+### When to Use This Helm Chart
+Use this Helm chart if you are doing any of the following:
+- Deploying either Nexus Repository Pro or OSS to an on-premises environment with bare metal/VM server (Node)
+- Deploying a single Nexus Repository instance within a Kubernetes cluster that has a single Node configured
+
+> **Note**: If you are using Nexus Repository Pro, your license file and embedded database will reside on the node and be mounted on the container as a Persistent Volume (required).
+
+
+### When Not to Use This Helm Chart
+Do not use this Helm chart and, instead, refer to our [resiliency documentation](https://help.sonatype.com/repomanager3/planning-your-implementation/resiliency-and-high-availability) if you are doing any of the following:
+
+- Deploying Nexus Repository Pro to a cloud environment with the desire for automatic failover across Availability Zones (AZs) within a single region
+- Planning to configure a single Nexus Repository Pro instance within your Kubernetes/EKS cluster with two or more nodes spread across different AZs within an AWS region
+- Using an external PostgreSQL database
+
+> **Note**: A Nexus Repository Pro license is required for our resilient deployment options. Your Nexus Repository Pro license file must be stored externally as either mounted from AWS Secrets/Azure Key Vault in AWS/Azure deployments or mounted using Kustomize for on-premises deployments (required).
+
+> **Note**: We do not currently provide Helm charts for our resilient deployment options.
+
+---
+
+## Prerequisites for This Chart
 
 - Kubernetes 1.19+
 - PV provisioner support in the underlying infrastructure
 - Helm 3
 
 ### With Open Docker Image
 
-By default, the Chart uses Sonatype's Public Docker image. If you want to use a different image, run with `--set nexus.imageName=<my>/<image>`.
+By default, this Chart uses Sonatype's Public Docker image. If you want to use a different image, run with the following: `--set nexus.imageName=<my>/<image>`.
 
 ### With Red Hat Certified container
 
-If you're looking run our Certified Red Hat image in an OpenShift4 environment there is a Certified Operator in OperatorHub.
+If you're looking run our Certified Red Hat image in an OpenShift4 environment, there is a Certified Operator in OperatorHub.
+
+---
```

Comment:
I previewed all of these horizontal lines in VSCode preview. Sometimes, it required space before/after the line to work properly. Other times, it required only space before or after. But, they all appear correctly spaced in preview.